deployment/nginx.coffee

```diff
@@ -15,6 +15,10 @@ createUpstreams = (KONFIG) ->
       servers += '\n' if servers isnt ''
       port = parseInt(port, 10)
 
+      if name is 'webserver' and options.kubernetes.ports?
```

Comment:
This is not the place where we should handle port swapping. Please pass a new flag to configure and do this in first step.